tools/koch/koch.nim

```diff
@@ -243,6 +240,15 @@ proc buildTools(args: string = "") =
       "--opt:speed --stacktrace -d:debug --stacktraceMsgs -d:nimCompilerStacktraceHints --excessiveStackTrace:off " & defineSourceMetadata() & " " & args,
       outputName = "nim_dbg")
 
+proc addTools(b: var Builder) =
+  let tools = [
```

Comment:
maybe a `b.buildGroup` call that takes a name and a bunch of `addxxx` calls, returning the name as an alias?